src/Redux/Countries/CountriesSlice.jsx

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please always make sure that you are not disabling the `linters` either *intentionally* or **unintentionally** because by doing this you are compromising the `quality` of the code, which as a developer is not a good practice and can cause problems, which will be difficult to debug at the later stages of development.